src/components/pages/landingpage/customers.tsx

```diff
@@ -66,23 +66,63 @@ const CustomerLogo = ({
   );
 };
 
+const AriaCustomerLogo = () => {
+  return (
+    <ul
+      style={{
+        position: 'absolute',
+        width: '1px',
+        height: '1px',
+        padding: '0',
+        margin: '-1px',
+        overflow: 'hidden',
+        clip: 'rect(0, 0, 0, 0)',
+        whiteSpace: 'nowrap',
+        border: '0',
+        listStyleType: 'none',
+      }}
+      aria-label={'Our Customers:'}
+    >
+      <li>Allianz</li>
+      <li>Rosenbauer</li>
+      <li>Media Markt</li>
+      <li>Saturn</li>
+      <li>Green City</li>
+      <li>FC Bayern</li>
+      <li>Zeiss</li>
+      <li>A.D.A.C</li>
+    </ul>
+  );
+};
+
 export const Customers = () => {
   return (
-    <LogosContainer>
-      <LogoLine>
-        <CustomerLogo desktopWidth={330} mobileWidth={127} show="allianz" />
-        <CustomerLogo desktopWidth={451} mobileWidth={176} show="rosenbauer" />
-      </LogoLine>
-      <LogoLine>
-        <CustomerLogo desktopWidth={403} mobileWidth={170} show="mediamarkt" />
-        <CustomerLogo desktopWidth={285} mobileWidth={121} show="saturn" />
-        <CustomerLogo desktopWidth={337} mobileWidth={156} show="greencity" />
-      </LogoLine>
-      <LogoLine>
-        <CustomerLogo desktopWidth={112} mobileWidth={48} show="fcbayern" />
-        <CustomerLogo desktopWidth={92} mobileWidth={39} show="zeiss" />
-        <CustomerLogo desktopWidth={324} mobileWidth={138} show="adac" />
-      </LogoLine>
-    </LogosContainer>
+    <>
+      <LogosContainer aria-hidden={true}>
+        <LogoLine>
+          <CustomerLogo desktopWidth={330} mobileWidth={127} show="allianz" />
+          <CustomerLogo
+            desktopWidth={451}
+            mobileWidth={176}
+            show="rosenbauer"
+          />
+        </LogoLine>
+        <LogoLine>
+          <CustomerLogo
+            desktopWidth={403}
+            mobileWidth={170}
+            show="mediamarkt"
+          />
+          <CustomerLogo desktopWidth={285} mobileWidth={121} show="saturn" />
+          <CustomerLogo desktopWidth={337} mobileWidth={156} show="greencity" />
+        </LogoLine>
+        <LogoLine>
+          <CustomerLogo desktopWidth={112} mobileWidth={48} show="fcbayern" />
+          <CustomerLogo desktopWidth={92} mobileWidth={39} show="zeiss" />
+          <CustomerLogo desktopWidth={324} mobileWidth={138} show="adac" />
+        </LogoLine>
+      </LogosContainer>
+      <AriaCustomerLogo />
```

Comment:
Please move that into the logos container, otherwise it's outside of the page layout and will end up at the top of the page. It's probably not a big issue for the screen reading itself but if you're only partially visually disabled you would have a jumping page

![CleanShot 2024-01-16 at 17 19 05@2x](https://github.com/satellytes/satellytes.com/assets/1701755/5c5d375b-ae8e-4974-b246-5f41b9e8a4fc)